docs/mp/health/01_introduction.adoc

```diff
@@ -114,24 +138,369 @@ microservice reports itself as ready.
 |Treats the instance as still starting up; does not check liveness or readiness until the startup probe reports success or times out according to its configuration.
 |===
 
-=== Built-in and Custom Health Checks
 
-==== Built-in Health Checks
-Helidon provides built-in, default checks for each endpoint.
-The built-in liveness checks include various environmental values, such as whether the JVM has detected deadlocks
-or whether there is sufficient heap space. The built-in readiness and startup checks always report `UP`.
+== Configuration
+
+Health checks may be configured using the following properties.
+
+The responsible for configuration class is:
```

Comment:
Not sure this is the right heading, but at least fixing english:
`The class responsible for configuration is:`